src/spider/storage/MysqlStorage.cpp

```
@@ -1,130 +1,1106 @@
+    KEY parent USING BTREE,
+    KEY child USING BTREE,
+    CONSTRAINT task_dep_parent FOREIGN KEY (parent) REFERENCES tasks (id) ON UPDATE NO ACTION ON DELETE CASCADE,
+    CONSTRAINT task_dep_child FOREIGN KEY (child) REFERENCES tasks (id) ON UPDATE NO ACTION ON DELETE CASCADE
+))";
+
+char const* const cCreateTaskInstanceTable = R"(CREATE TABLE IF NOT EXISTS task_instances (
+    id BINARY(16) NOT NULL,
+    task_id BINARY(16) NOT NULL,
+    start_time TIMESTAMP NOT NULL,
+    CONSTRAINT instance_task_id FOREIGN KEY (task_id) REFERENCES task (id) ON UPDATE NO ACTION ON DELETE CASCADE,
+    PRIMARY KEY (id)
```

cppcheck

[performance] Variable 'm_id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)